### PR TITLE
Fixes #18461 - reword 'External groups'

### PR DESCRIPTION
**Pull request by shlomizadok**

### CODE REVIEW

File: app/views/usergroups/_form.html.erb [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <li class="active"><a href="#primary" data-toggle="tab"><%= _("User group") %></a></li>
```
Comment (tbrisker) [inline app/views/usergroups/_form.html.erb]: How about fixing the case here as well while you're at it?

Comment (shlomizadok) [reply app/views/usergroups/_form.html.erb]: Done :+1:

**Comment by tbrisker** [comment]
APJ

**Comment by tbrisker** [comment]
thanks @shlomizadok !

**Pull request merged**